assets/airports/sawh.json

```diff
@@ -0,0 +1,149 @@
+    "LOKEN": ["S53.40832343751806", "W68.42136599380468"],
+    "PWL":   ["S54.92948064936479", "W67.62114014089683"],
+    "UGIRA": ["S54.50004733240834", "W68.31783815953314"],
+    "USU":   ["S54.83806563867309", "W68.28420167323435"],
+    "WH213": ["S54.73004760984458", "W67.64228882299615"],
+    "WH214": ["S54.79640254056611", "W67.63503784627639"],
+    "WH321": ["S54.91970022935944", "W68.34190737392117"],
+    "WH322": ["S54.92415668876073", "W68.05337892529478"],
+    "WH323": ["S54.92583496759332", "W67.90886293095660"],
+    "WH324": ["S54.92664514611024", "W67.76444764462840"],
+    "WH325": ["S54.76166982464822", "W67.72698426491150"],
+    "WH326": ["S54.87192146789932", "W68.04139467210577"]
+  },
+  "runways": [
+    {
+      "name": ["7", "25"],
+      "name_offset": [[0, 0], [0, 0]],
+      "end": [["S54d50m39.12", "W68d19m3.09"], ["S54d50m32.72", "W68d16m20.17"]],
+      "ils": [false, true]
+    }
+  ],
+  "sids": {
+    "USU2A": {
+      "icao": "USU2A",
+      "name": "Oosuaia Two Alpha",
+      "rwy": {
+        "7" : ["_RWY07", ["_USU14DME093", "A45+|S250"]]
+      },
+      "transitions": {
+        "DARIN": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "DARIN", "ALMIR"],
+        "KEXOP": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "KEXOP", "GRA"],
+        "LITOK": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "LITOK", "ALMIR"],
+        "UGIRA": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "UGIRA", "LOKEN"],
+        "GRA":   [["_USU14DMEGRA", "S250"], "GRA"]
+      },
+      "draw": [["_RWY07", "_USU14DME093", "_USU14DME110", "USU"], ["_RWY25", "_RWY25R070", "_USU07DME093"], ["USU", "DARIN*"], ["USU", "KEXOP*"], ["USU", "LITOK*"], ["USU", "UGIRA*"], ["_USU14DME093", "_USU14DMEGRA", "GRA*"]]
+    },
+    "USU2B": {
+      "icao": "USU2B",
+      "name": "Oosuaia Two Bravo",
+      "rwy": {
+        "25" : [["_RWY25", "A4+"], "_USU07DME093", ["_USU14DME093", "A45+|S250"]]
+      },
+      "transitions": {
+        "DARIN": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "DARIN", "ALMIR"],
+        "KEXOP": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "KEXOP", "GRA"],
+        "LITOK": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "LITOK", "ALMIR"],
+        "UGIRA": [["_USU14DME110", "S250"], ["USU", "A80+|S250"], "UGIRA", "LOKEN"],
+        "GRA":   [["_USU14DMEGRA", "S250"], "GRA"]
+      },
+      "draw": [["_RWY25", "_RWY25R070", "_USU07DME093"]]
+    },
+    "PWL3": {
+      "icao": "PWL3",
+      "name": "Williams Three",
+      "rwy": {
+        "25" : [["_RWY25", "A4+|S180"], ["WH321", "A17+|S180"], ["WH322", "A50+|S180"], ["WH323", "A75+|S180"], ["WH324", "A80+|S180"]]
+      },
+      "transitions": {
+        "USU": [["WH326", "A110+|S180"], ["USU", "S180"], "UGIRA", "LOKEN"],
+        "USU": [["WH326", "A110+|S180"], ["USU", "S180"], "KEXOP", "GRA"],
+        "USU": [["WH326", "A110+|S180"], ["USU", "S180"], "LITOK", "ALMIR"],
```

Comment:
Only this USU transition is used, it overwrites the previous ones.  I don't think that's the intent, they should probably be renamed based on the last fix.